journal/modernized/2-journal.adoc

```diff
@@ -7000,10 +7018,16 @@ Open infidelity is easier borne with, than when combined with dissimulation,
 and does less mischief; because, by the latter the simple are led astray,
 many of whom are to be pitied, and for whom I feel a tender regard.`"
 
+--
+
 [.offset]
 The following address appears to belong to this period, though it is without date,
 and may with propriety be introduced here:
 
+[.embedded-content-document.address]
+--
+//maybe this should be an epistle?
```

Comment:
Yeah, it's close, but I think we should change it to epistle. Even though he uses the word "address" it's definitely to Friends, which is why I think you left this comment. Good catch, let's make the change.